<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisType</class>
 <widget class="QDialog" name="AnalysisType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>252</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>252</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Control the golden section method and analytical solution</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Analytical solution type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Initial excess pore pressure p0 (kPa)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Number of calculation steps</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Time between calculation step (s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Initial guess Cd0 factor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Tolerance of the golden section method</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="typeCombo">
         <item>
          <property name="text">
           <string>Equal strain, no smear zone (Hansbo)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Equal strain, with smear zone (Hansbo)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Equal strain, with smear zone, with well resistance (Hansbo)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Free strain, no smear zone (Barron)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Free strain, with smear zone (Barron)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="p0Line">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="numberOfStepLine">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dtLine">
         <property name="text">
          <string>86400</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cd0Line">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tolLine">
         <property name="text">
          <string>0.001</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Find Cd between                                     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="startLine">
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>and</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="endLine">
       <property name="text">
        <string>2.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Note: The first step is always undrained analysis (dt=0)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
